src/db/mysql-connector.js

```diff
@@ -1,9 +1,17 @@
 'use strict';
-import * as mysql from 'mysql';
+import * as Sequelize from 'sequelize';
 
-export const connectionPool = mysql.createPool({
-    host: process.env.DATABASE_HOST,
-    port: process.env.DATABASE_PORT,
-    user: process.env.DATABASE_USER,
-    password: process.env.DATABASE_PASSWORD,
+export const sequelize = new Sequelize(process.env.DATABASE_NAME,
+    process.env.DATABASE_USER, process.env.DATABASE_PASSWORD, {
```

Comment:
I can see some indentation issues here..

Reply:
yeah I noticed. But shouldn't fields inside the bracers need to be indented by one?